ddtrace/llmobs/_integrations/anthropic.py

```diff
@@ -150,16 +150,20 @@ def _extract_output_message(self, response):
 
         elif isinstance(content, list):
             for completion in content:
+                tool_calls_info = []
```

Comment:
is there a need to instantiate the list here if we're only going to handle it in the tool call block? (i.e. line 158)